server/src/Position.ts

```diff
@@ -0,0 +1,15 @@
+export default class Position {
+    x: number;
+    y: number;
+    z: number;
+
+    constructor(x: number = 0, y:number = 0, z: number = 0) {
```

Comment:
```suggestion
    constructor(public x: number = 0, public y: number = 0, public z: number = 0) { }
```

By setting a constructor param as `private` or `public`, this automatically initialises the correct member.

Reply:
Done.